File: AD-HOC/2868/main.dart
import 'dart:io';

int eval(String expression) {
  var [a, operator, b] = expression.trim().split(' ').toList();
  if (operator == '+') return int.parse(a) + int.parse(b);
  if (operator == '-') return int.parse(a) - int.parse(b);
  return int.parse(a) * int.parse(b);
}

void main() {
  int c = int.parse(stdin.readLineSync()!);
  for (int times = 0; times < c; times++) {
    var [expression, result] = stdin.readLineSync()!.split('=').toList();
    int difference = (eval(expression) - int.parse(result)).abs();
    print('E' + 'r' * difference + 'ou!');
  }
}

File: AD-HOC/2868/main.js
let input = require('fs').readFileSync('/dev/stdin', 'utf-8');
let lines = input.split('\n');
let c = parseInt(lines[0]);
for (let i = 1; i <= c; i++) {
    let [expression, result] = lines[i].split('=');
    let difference = Math.abs(eval(expression.replace('x', '*')) - parseInt(result));
    console.log('E' + 'r'.repeat(difference) + 'ou!');
}